### ISSUE TITLE
Weird output for some functions

### ISSUE THREAD

**Issue by benny-z** [google/latexify_py#23]
There are some cases where generated LaTeX contains some weird strings (probably artifacts of the _ast_ library). Here's a minimal example:
```python
@latexify.with_latex
def foo(a):
 c = foo(a-1)
 return c

>>> print(foo)
>>> \mathrm{foo}(a)\triangleq <_ast.Assign object at 0x000001E3DD084F88>
```

**Comment by yixingfu** [comment]
```
@latexify.with_latex
def foo(a):
    return foo(a-1)
```
works fine. I think the problem is `visit_Assign()` not implemented.

**Comment by odashi** [comment]
Yes, the source of the problem is that we do not implement `visit_Assign` yet (because handling multiple statements is not trivial #7 ). I guess we need to put `NotImplementedError` so far.

**Comment by 1MLightyears** [comment]
I can think of two ways to cover that:
 - treat assigns as inline/macro-like statements, and replace them during handling "return" statement;
 - output multiple lines of latex codes, while each line covers one assign/"return" statement;

Which one do you prefer? Or anyone has better ideas?

------

(to other viewers:)If latexify is going to support multiple statements, the logic is sure going to be really complex. Can never be too cautious to make the decision.

**Comment by odashi** [comment]
Solved by #51
